src/component/handlers/edit/editOnCompositionStart.js

```diff
@@ -18,9 +18,10 @@ var EditorState = require('EditorState');
  * The user has begun using an IME input system. Switching to `composite` mode
  * allows handling composition input and disables other edit behavior.
  */
-function editOnCompositionStart(): void {
+function editOnCompositionStart(e): void {
   this.setRenderGuard();
   this.setMode('composite');
+  this._onCompositionStart(e);
```

Comment:
Right. `compositionstart` is used as a flag to indicate that the editor should switch to `composite` mode. This swaps in `DraftEditorCompositionHandler` as the editor's handler bundle, and all subsequent events are handled there until the editor switches back to `edit` mode. So you shouldn't need to add any code here.